couchbase/admin/bucket/v1/bucket.proto

```diff
@@ -54,7 +54,7 @@ message ListBucketsResponse {
   message Bucket {
     string bucket_name = 1;
     bool flush_enabled = 2;
-    uint64 ram_quota_bytes = 3;
+    uint64 ram_quota_mb = 3;
```

Comment:
uint64 is a lot of mebibytes :-D

Reply:
Ah, but it's more or less aligned with what the JVM SDKs are already representing this field, so no problem. Sorry for the noise.